packages/components/src/autocomplete/autocompleter-ui.native.js

```diff
@@ -69,9 +69,6 @@ export function getAutoCompleterUI( autocompleter ) {
 			} else if ( isVisible && text.length === 0 ) {
 				startAnimation( false );
 			}
-			// Temporarily disabling exhaustive-deps to avoid introducing unexpected side effecst.
-			// See https://github.com/WordPress/gutenberg/pull/41820
-			// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure about that. The comment made sense when we were ignoring the ESLint error, but doesn't make sense anymore IMHO. If you take a look at the rest of the instances, you'll see that I've:

- Preserved comments that add extra context on top of the sole "make exhaustive deps checks pass"
- Removed comments that only relate to "making exhaustive deps checks pass" since those are no longer relevant IMHO.

Let me know what you think.

Reply:
Makes sense I thought that, in this case, linking back to https://github.com/WordPress/gutenberg/pull/41820 could be valuable, since it shows a previous attempt at adding dependencies that ended up being reverted — I thought that providing such context could be useful to a future reader.

In general I agree with your approach.

Reply:
Sounds good. I'll work on retaining those comments that have context then.